charts/aws-ebs-csi-driver/templates/node/daemonset-windows.yaml

```diff
@@ -1,28 +1,29 @@
 {{- if .Values.node.enableWindows }}
-kind: DaemonSet
+# Node Service
 apiVersion: apps/v1
+kind: DaemonSet
 metadata:
-  name: ebs-csi-node-windows
+  name: {{ include "aws-ebs-csi-driver.node-windows.fullname" . }}
   labels:
-    {{- include "aws-ebs-csi-driver.labels" . | nindent 4 }}
+    {{- include "aws-ebs-csi-driver.node-windows.labels" . | nindent 4 }}
 spec:
   selector:
     matchLabels:
-      app: ebs-csi-node
-      {{- include "aws-ebs-csi-driver.selectorLabels" . | nindent 6 }}
+      {{- include "aws-ebs-csi-driver.node-windows.selectorLabels" . | nindent 6 }}
+  {{- with .Values.node.updateStrategy }}
   updateStrategy:
-    {{ toYaml .Values.node.updateStrategy | nindent 4 }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
   template:
     metadata:
       labels:
-        app: ebs-csi-node
-        {{- include "aws-ebs-csi-driver.labels" . | nindent 8 }}
-        {{- if .Values.node.podLabels }}
-        {{- toYaml .Values.node.podLabels | nindent 8 }}
-        {{- end }}
+        {{- include "aws-ebs-csi-driver.node-windows.labels" . | nindent 8 }}
+      {{- with .Values.node.podLabels }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
       {{- with .Values.node.podAnnotations }}
       annotations:
-        {{- toYaml . | nindent 8 }}
```

Comment:
Indents like this are to make the template more readable and show where the result will be.  I would leave them in.